evm/computation.py

```diff
@@ -303,6 +304,11 @@ def get_gas_remaining(self):
         else:
             return self.gas_meter.gas_remaining
 
+    @contextmanager
+    def state_db(self, read_only=False):
```

Comment:
Note: #405 split `state_db` into `read_only_state_db` and `mutable_state_db`.
I can wait for this one to get merged first and make consequent updates.